disclosure_backend/settings.py

```diff
@@ -1,7 +1,8 @@
 import os
+import os.path as op
```

Comment:
I think I speak for @adborden too when I say that we have relatively minimal experience with python idioms, so if these abbreviations are common among many python projects, we can learn them.